src/DatabaseSchema.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
It's kind of crowded inside `src` already. Let's move these new classes to a new directory, please... 
Maybe `src/app/Library/Database/*`?